docs/sql-getting-started.md

```diff
@@ -41,7 +41,7 @@ The entry point into all functionality in Spark is the [`SparkSession`](api/java
 
 <div data-lang="python"  markdown="1">
 
-The entry point into all functionality in Spark is the [`SparkSession`](api/python/pyspark.sql.html#pyspark.sql.SparkSession) class. To create a basic `SparkSession`, just use `SparkSession.builder`:
+The entry point into all functionality in Spark is the [`SparkSession`](api/python/reference/api/pyspark.sql.SparkSession.html) class. To create a basic `SparkSession`, just use `SparkSession.builder`:
```

Comment:
Hi, @dhruvildave . Is this correct? In Apache Spark website, the original one seems to work correctly.
- https://spark.apache.org/docs/latest/sql-getting-started.html

Reply:
Hello! Thanks for the review. I have created a small video showing exactly where the errors are encountered.

https://youtu.be/wIZ56za9zmA

Reply:
Oh, thanks!

Reply:
Ah, got it. It's inside Python tab. I missed the point.